src/components/Calculator.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] I would not recommend disabling linters instead, try improving the code. Here you have to check the prop types, It is helpful as it would check whether the coming prop is of the type which we are expecting or not.